### DISCUSSION TITLE
Photos taken with an iPad rotated horizontally, do not retain the rotation and are turned vertically.

### DISCUSSION THREAD

**Discussion by ranleu** [opengisch/QField#5870] [Bugs]
One more bug related to iPad rotataion. Images take horizontally do not retain the rotation and are turned vertically. 
![D2_04333_O_007_KW42_20241015-155704703](https://github.com/user-attachments/assets/cefa97b7-1fb6-4b32-86d7-752e2e9c3dad)

**Comment by Growth64** [comment]
I have the same probleme with Android
